api/src/lib/interface.ts

```diff
@@ -101,8 +101,8 @@ export interface GetPublicationPathParams {
     id: string;
 }
 
-export interface GetLinksQueryParams {
-    publicationID: string
+export interface GetPublicationQueryParams {
```

Comment:
Maybe "GetSeedDataPublicationsFilters" would be more suitable name for this. It's still long, but at least it's specific.
You can also add the "userId" here so we can see exactly all the filters applied to the query. Because atm the userId is hardcoded in the service of this route and that's not ideal.

Finally, your route would look something like this: /seed-data-publications?title={publicationTitle}&userId={userId}

Reply:
I've changed the name of the interface to the one suggested, but no worried about the userId, it's only ever octopus that will have seed data, so don't need to adapt for other users.